packages/devextreme/testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/aiDialog.tests.js

```diff
@@ -0,0 +1,146 @@
+import $ from 'jquery';
+import AIDialog from '__internal/ui/html_editor/ui/aiDialog';
+import { isPromise } from 'core/utils/type';
+
+import 'ui/menu';
+import 'ui/popup';
+import 'ui/text_area';
+import 'ui/select_box';
+
+const AI_DIALOG_CONTENT_CLASS = 'dx-aidialog-content';
+const AI_DIALOG_CONTROLS_CLASS = 'dx-aidialog-controls';
+const DIALOG_CLASS = 'dx-formdialog';
+
+const moduleConfig = {
+    beforeEach: function() {
+        this.$element = $('#htmlEditor');
+        this.componentMock = {
+            _createComponent: ($container, Widget, options) => {
+                return new Widget($container, options);
+            },
+            $element: () => this.$element,
+        };
+    }
+};
+
+QUnit.module('AIDialog', moduleConfig, () => {
+    QUnit.test('Should render AI dialog content', function(assert) {
+        const aiDialog = new AIDialog(this.componentMock, {}, { container: this.$element });
+
+        const commandsMap = {
+            translate: {
+                name: 'translate',
+                text: 'Translate',
+                options: ['english', 'german']
+            }
+        };
+
+        const payload = {
+            currentCommand: 'translate',
+            currentCommandOption: 'english',
+            commandsMap,
+            text: 'Test text',
+        };
+
+        aiDialog.show(payload);
+
+        const $wrapper = this.$element.find(`.${DIALOG_CLASS}`);
+        const $aiContent = $wrapper.find(`.${AI_DIALOG_CONTENT_CLASS}`);
+        assert.strictEqual($aiContent.length, 1, 'AI dialog content rendered');
+
+        const $controls = $aiContent.find(`.${AI_DIALOG_CONTROLS_CLASS}`);
+        assert.strictEqual($controls.length, 1, 'Controls container rendered');
+
+        const $selectBoxes = $aiContent.find('.dx-selectbox');
```

Comment:
let's use constants here an in lot of other cases (`dx-menu` `dx-menu-item` etc)